### PR TITLE
Add lovelace cards to readme

### PR DESCRIPTION
**Pull request by DurgNomis-drol**
Add lovelace cards to readme.

Reasoning is that it will be easier for the user find a card that is design for this integration.

### CODE REVIEW

**Comment by KapJI** [comment]
Not related but can use please also update link to logo image?

**Comment by DurgNomis-drol** [comment]
I was just thinking about @KapJI and yes! 😆

**Comment by leikoilja** [comment]
That sounds good to me! Do you mind adding a small remark saying something like: "open a PR to add your card" sort of vibe? I think it wouldn't hurt adding some good community lovelace cards here, for that same reason as you mentioned? :)

File: README.md [hunk 1]
```diff
@@ -184,6 +190,16 @@ ln -s ha-google-home/custom_components/ha-google-home ~/.homeassistant/custom_co
 
 Make sure that you have your Home Assistant Container network set to 'host', as perscribed in the official docker installation for Home Assistant.
 
+## Lovelace Cards
+
+<b>Open a PR to add your card here!</b>
```
Comment (KapJI) [inline README.md:195]: Please use markdown :)
```suggestion
**Open a PR to add your card here!**
```

**Review by leikoilja — APPROVED** [review]
Thank you both 💥

**Pull request merged**